packages/eslint-config-sentry-app/index.js

```diff
@@ -31,6 +31,17 @@ module.exports = {
   settings: {
     'import/resolver': 'webpack',
     'import/extensions': ['.js', '.jsx'],
+    'import/no-restricted-imports': [
```

Comment:
@billyvg I don't think `import/no-restricted-imports` is a rule from the `eslint-plugin-import` plugin.

Looks like it's an first-class eslint rule: https://eslint.org/docs/rules/no-restricted-imports

Reply:
Oops good catch